### ISSUE TITLE
Search does not find apps after upgrade to Ubuntu Mate 20.04.1

### ISSUE THREAD

**Issue by juzi** [ubuntu-mate/mate-menu#80]
I've upgraded my ubuntu mate from 18.04.3 to 20.04.1 and something must have gone wrong with the Advanced Mate Menu.
I can't see any applications in the menu and when I want to use quick search it does not fine any applications either. 
I googled a lot but couldn't find how I can completely reset the menu settings and the application list/index. 
Funny enough, I see the applications if I right-click and go to "Edit menu" (see screenshot).

https://i.imgur.com/eNUX6d7.png
https://i.imgur.com/wCYcTH2.png

thanks in advance.

**Comment by vkareh** [comment]
Hi @juzi - try a right-click on the menu and select "Reload plugins". If that doesn't work, try instead "Remove From Panel". The you can run this command in a terminal:
```
gsettings reset-recursively org.mate.mate-menu
```
This will reset all options in the menu to their default state, then you can re-add it to the panel.

Let me know if this works or not and we'll debug more from there. Good luck!

**Comment by juzi** [comment]
Hi,
thanks for the quick reply! 
Unfortunately, your suggestions did not help.
Another thing I had tried this morning is copying all *.desktop files from /usr/share/applications/ to my ~/.local/share/applications/
But that also didn't change anything.

**Comment by juzi** [comment]
Today I was very desperate and did a 
  dconf reset -f /
which completely reset my desktop. 
Still, Advanced Mate Menu does not show applications. 
The Brisk Menu Launcher does show all applications. I guess I'm just gonna use that one until I perform a fresh install of Ubuntu Mate in a few years time.

**Comment by vkareh** [comment]
ugh, I sent a reply yesterday and it seems like it didn't get through... 

>  Another thing I had tried this morning is copying all *.desktop files from /usr/share/applications/ to my ~/.local/share/applications/

Did you run `sudo update-desktop-database`?

You might also want to try uninstalling the mate-menu and reinstalling it to see if that helps. Sorry, I've never encountered (nor seen anyone report) this issue and I'm not even sure how to reproduce :(

**Comment by juzi** [comment]
I tried the update-desktop-database command (which I had never heard of before) and got some errors:
--------------------
Could not parse file "/usr/share/applications/screensavers/glitchpeg.desktop": Key file contains line ?several times a second.  After a while, finds a new image to corrupt. Written by Jamie Zawinski; 2018.? which is not a key-value pair, group, or comment
Error in file "/usr/share/applications/org.kde.kdeconnect_open.desktop": "*/*" is an invalid MIME type ("*" is an unregistered media type)
--------------------
I simply removed those files and then the command ran without errors. However, still no applications in mate-menu.
Then I purged mate-menu (sudo apt purge mate-menu) and installed it again. Still no applications visible. 

I also haven't seen this behaviour before. I think I'll give up on this issue.
I really appreciate your help and the time you spent answering to my issue!  
Don't worry, I'm just going to do a fresh install at some point. 
Thanks again
